### ISSUE TITLE
Add Teamspeak support

### ISSUE THREAD

**Issue by Arcanum417** [antonpup/Aurora#568]
Speaking indicator
Muted mic/audio status
AFK status
Someone else speaking
Highlight PTT key and other usefull binds
Recording status
Unread message indicator (gradient for more messages)
Poke indicator
Talk power status

**Comment by brainbug89** [comment]
Should be possible with the [TS3 Plugin SDK](https://www.teamspeak.com/teamspeak3sdk.html)

**Comment by simon-wh** [comment]
Yeah, I've looked into it and a lot of the suggestions will be possible through the ClientQuery plugin. However, I want to rework Overlays to be more in line with what we already have for Applications with Layers + Profiles before I finish the support. Currently, I'm thinking of having the Overlays act like any other Application except that are displayed based on the process running instead of being the foreground and using Layer Logic to trigger effects when certain conditions are met. I can see this working quite well for the Skype overlay, but I'm unsure about what to do with the Volume Overlay, I might include something like a Windows overlay that includes the Volume overlay and other possibilities. Any feedback would be appreciated for this concept.

**Comment by Dlaw2017** [comment]
I think that's a great idea! Being able to see that I have Skype and TS messages while in game would be very nice. I see it working well for a lot of things, like my custom shortcut assistant layers that I have to copy one by one to new profiles that I want to have them in.

Maybe we could even be able to setup a hotkey to show things like that? This way, it's not always in our face, but we have the information available without alt-tabbing.

**Comment by FrozenBeard** [comment]
This is a great idea!

**Comment by DevJPM** [comment]
Now with Overlay support being added, I'm currently looking into writing a helper application for Teamspeak based on the ClientQueryPlugin's Telnet interface and then integrating a handler into Aurora (similar to how Skype works). Though admittedly I haven't found Skype yet in the Aurora UI, so I might need help for the final stretches - as well as for finding / identifying interesting default patterns from Teamspeak's data - as I'm mostly interestedin having access to the variables and having an automatically managed helper process.
